### PR TITLE
My first Code Review

### PR DESCRIPTION
**Pull request by bhskr44**
This repository was added during the microverse trails and is now a code review on that project.

### CODE REVIEW

File: fcc-test-suite-template-master/index2.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/styles.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"></img>
```
Comment (bhskr44) [inline fcc-test-suite-template-master/index2.html:13]: Please Note: External images might be under copyright. If you do not get permission to use it, you may be in violation of copyright laws. In addition, you cannot control external images; they can suddenly be removed or changed.

Therefore, I would suggest using local images in place of the external link.

File: fcc-test-suite-template-master/css/styles.css [hunk 2]
```diff
@@ -1,48 +1,53 @@
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 100px;
+}
+
+@media only screen and (min-width: 600px) {
+  body {
+    background-color: blue;
+  }
```
Comment (bhskr44) [inline fcc-test-suite-template-master/css/styles.css:52]: I will suggest that you change the background color to white for better readability.

**Review by bhskr44 — COMMENTED** [review]
2mk8se


# Changes Required ♻️ 


You're almost there,
There are just a few changes left.
check my comments below and make the required changes.

Happy Coding 👋

File: fcc-test-suite-template-master/css/styles.css (hunk 2 above)
Comment (sajid-munawar) [reply fcc-test-suite-template-master/css/styles.css:52]: - [ ] this is a very nice review you have done, polite communication, and nice explanation. keep it up.

File: fcc-test-suite-template-master/index2.html (hunk 1 above)
Comment (sajid-munawar) [reply fcc-test-suite-template-master/index2.html:13]: - [x] You did well here, please make sure you explained a bit more why you suggested the changes. if you give more explanation the student will understand better where the bugs are and how to fix them.

**Review by sajid-munawar — COMMENTED** [review]
# Changes Required 
 ### Hi @bhskr44 
 Nice job, you are almost there.
 
 
 
 ## (Highlights) Good Points: 
 * Good commit messages. :heavy_check_mark:
 * No linter errors. :heavy_check_mark:
 * Followed Github flow :heavy_check_mark:
 * Professional README file. :heavy_check_mark:
 * PR has a good title and summary :heavy_check_mark:

 ## (Changes Required) Aspects to improve: 
 * [ ]  Please check my inline comments.
 
 ## [Optional] suggestions:
 * Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 **Cheers and Happy coding! :clap::clap::clap:**
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear
 Please, remember to tag @sajid-munawar in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by bhskr44 — COMMENTED** [review]
download


Great Doing on implementing those changes. 👍 
I had added clarification about why to add a local image in place of the external link. I suggest you go through the details and check if you could make the necessary changes.



Happy Coding 👨‍💻

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @bhskr44 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**